### PR TITLE
Semaphore v2 migration / pin update process addition

### PR DESCRIPTION
**Pull request by Brian-McM**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
OK, I will not merge the pull request when it's ready.

File: .semaphore/semaphore.yml
Comment (doublek) [inline .semaphore/semaphore.yml:56]: nit: Needs a newline.

**Review by doublek — APPROVED** [review]
LGTM - one minor nit though.

File: .semaphore/semaphore.yml
Comment (Brian-McM) [reply .semaphore/semaphore.yml]: done

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
